.circleci/config.yml

```diff
@@ -121,3 +135,6 @@ workflows:
           filters:
             branches:
               only: master
+      - github_release:
+          requires:
+            - production_release
```

Comment:
nitpick: noticed that the name for this step is release in other repos & it is production_release on this one.

Worth changing, to keep things in line.

Otherwise 👍